src/main/java/ladder/contoller/InputView.java

```diff
@@ -0,0 +1,12 @@
+package ladder.contoller;
+
+import java.util.Scanner;
+
+public class InputView {
```

Comment:
```suggestion
package ladder.view;

import java.util.Scanner;

public class InputView {
```

제 의도와 다릅니다. `ladder.view` 패키지로 원래대로 복구해주세요